operator/main.go

```diff
@@ -42,9 +47,20 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
+	var (
+		awsRegion            string
+		chartRepository      string
+		gracePeriod          time.Duration
+		metricsAddr          string
+		namespace            string
+		enableLeaderElection bool
+	)
+
+	flag.StringVar(&awsRegion, "aws-region", "", "The AWS region where the operator's chart repository is hosted")
+	flag.StringVar(&chartRepository, "chart-repository", "", "The URI of the chart repository used by the operator")
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&namespace, "namespace", "default", "The cluster namespace where the operator will deploy releases")
```

Comment:
Why do we need to be explicit about this? Can't we just use `default` unless it's specified in the chart?

Reply:
AFAIK charts don't specify the namespace they should be deployed in. That's a decision we make at install time. We could add a namespace field to the `HelmReleaseSpec` if we wanted, but otherwise this is the only other place we choose which namespace to deploy in.